Correciones de estilo y ortografía hasta línea 186

nicolaslarrea opened the pull request.

nicolaslarrea committed:
Correciones de estilo y ortografía hasta línea 186

fauno commented:
maxpowergit notifications@github.com writes:

> You can merge this Pull Request by running:

que hace maxpower haciendo el grueso y luego ajustando D:

## 

http://librevpn.org.ar

nicolaslarrea commented:
y es depende cuanta atención pongo.

On 15/10/14 11:38, fauno wrote:

> maxpowergit notifications@github.com writes:
> 
> > You can merge this Pull Request by running:
> 
> que hace maxpower haciendo el grueso y luego ajustando D:
> 
> ## 
> 
> http://librevpn.org.ar
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/edsl/articulos/pull/42#issuecomment-59215404.

nicolaslarrea committed:
correciones ortográficas y de estilo hasta 748

volví a cagar la tabulación.

nicolaslarrea committed:
correciones ortográficas y de estilo hasta 844

nicolaslarrea committed:
Ahora si estaría para que lo revisen

Falta una foto que tampoco está en la página original, es un cuadro al que refiere el texto, pero no está la imagen. Habría que pedirselo a maxigas?
